src/app/pages/component-viewer/component-viewer.ts

```diff
@@ -90,6 +102,18 @@ export class ComponentBaseView implements OnInit, OnDestroy {
         this.tableOfContents.resetHeaders();
       }
     });
+
+    this.showToc.pipe(
+      skip(1),
+      takeUntil(this.destroyed)
+    ).subscribe(() => {
+      if (this.tableOfContents) {
+        this.viewers.forEach(viewer => {
+          viewer.contentRendered.emit(viewer._elementRef.nativeElement);
+        });
+        this.tableOfContents.updateScrollPosition();
```

Comment:
I don't think that this line is required since `contentRendered` triggers `updateTableOfContents()` or `updateScrollPosition()` in all cases. This just causes it to be called twice.

Reply:
I removed that line.